### PR TITLE
Adds important admonition and note to disconnected installations when…

### PR DESCRIPTION
**Pull request by stevsmit**
Adds important admonition and note to disconnected installations when using oc adm catalog and oc image mirror commands

Version(s):
4.10+

Issue:
https://issues.redhat.com/browse/OSDOCS-3907

Link to docs preview:

- https://51305--docspreview.netlify.app/openshift-enterprise/latest/installing/disconnected_install/installing-mirroring-installation-images.html#olm-mirror-catalog_installing-mirroring-installation-images
- https://51305--docspreview.netlify.app/openshift-enterprise/latest/installing/disconnected_install/installing-mirroring-installation-images.html#installation-mirror-repository_installing-mirroring-installation-images

Change management acks: 
QE: Ying Zhou
DPM: Kathryn Alexander 
PM: Daniel Messer
SME: Filip Brychta
PE: Eric Rich

Associated pull requests for Release Note changes and admonitions: 

4.12: https://github.com/openshift/openshift-docs/pull/51308
4.11: https://github.com/openshift/openshift-docs/pull/51309
4.10: https://github.com/openshift/openshift-docs/pull/51311

### CODE REVIEW

File: modules/installation-mirror-repository.adoc [hunk 1]
```diff
@@ -174,6 +174,11 @@ $ oc image mirror -a ${LOCAL_SECRET_JSON} --from-dir=${REMOVABLE_MEDIA_PATH}/mir
 ----
 +
 <1> For `REMOVABLE_MEDIA_PATH`, you must use the same path that you specified when you mirrored the images.
++
+[NOTE]
+====
+Due to the inclusion of old images in some image indexes, running `oc image mirror` might result in the following error: `error: unable to retrieve source image`. As a temporary workaround, you can pass in the `--continue-on-error` option to bypass the error and continue downloading the image index. For more information, see link:https://access.redhat.com/solutions/6975305[Service Mesh Operator mirroring failed].
```
Comment (kcarmichael08) [inline modules/installation-mirror-repository.adoc:180]: In the other file you say "you can *use* the `--continue-on-error` option" which I think is more clear than "pass in"

**Review by kcarmichael08 — COMMENTED** [review]
Left one comment below. Also, I think olm-mirroring-catalog.adoc may need a :_content-type: line and both files seem to need an abstract per https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc 

Also - one is included in the "important" chunk and the other is a "note" - not sure if that is intentional but I thought I would point it out.

This looks good though!

File: modules/installation-mirror-repository.adoc (hunk 1 above)
Comment (stevsmit) [reply modules/installation-mirror-repository.adoc]: Fixed!

Comment (stevsmit) [reply modules/installation-mirror-repository.adoc]: Fixed. Added the content type.

**Comment by kalexand-rh** [comment]
I approve both this change and the related release notes changes.

**Comment by dmesser** [comment]
I think what we want here is to use the `--skip-missing` flag instead of the broader `--continue-on-error`. The latter will also continue on other errors (like missing layers in the target mirror that were expected to be there) that the user will want to know about.

File: modules/installation-mirror-repository.adoc [hunk 2]
```diff
@@ -174,6 +174,11 @@ $ oc image mirror -a ${LOCAL_SECRET_JSON} --from-dir=${REMOVABLE_MEDIA_PATH}/mir
 ----
 +
 <1> For `REMOVABLE_MEDIA_PATH`, you must use the same path that you specified when you mirrored the images.
++
+[IMPORTANT]
+====
+Due to the inclusion of old images in some image indexes, running `oc image mirror` might result in the following error: `error: unable to retrieve source image`. As a temporary workaround, you can use the `--skip-missing` option to bypass the error and continue downloading the image index. For more information, see link:https://access.redhat.com/solutions/6975305[Service Mesh Operator mirroring failed].
```
Comment (sferich888) [inline modules/installation-mirror-repository.adoc:180]: Is **inclusion** the right word here. 
I get what your trying to say; we 'included' something we shouldn't have, but I am not 100% sure that will be clear to the reader who isn't well versed in what causes this issue. 

Is the following more informative (or too wordy)? 
```
Some image indexes include image references to images that no longer exist (on the registry). These images may have been pulled for various reasons, however the image index includes them; primarily to allow customers running those images an upgrade path off of that point to newer points on the upgrade graph, thus its possible that those mirroring some image indexes (with `oc image mirror`) will need to use `--skip-missing` images to avoid `error: unable to retrieve source image` errors, seen by these situations. For more information, see link:https://access.redhat.com/solutions/6975305[Service Mesh Operator mirroring failed].
```

**Review by mjpytlak — COMMENTED** [review]
The content LGTM @stevsmit. Trying to verify the published output in assemblies that share `modules/installation-mirror-repository.adoc`. Please drop a preview link in.

**Comment by mjpytlak** [comment]
Thanks for the additional preview link @stevsmit. Peer review LGTM.
